daft/runners/ray_runner.py

```diff
@@ -478,53 +479,75 @@ def build_partitions(
 # Give the same function different names to aid in profiling data distribution.
 
 
+@ray_tracing.RayFunctionWrapper.wrap
 @ray.remote
 def single_partition_pipeline(
+    job_id: str,
```

Comment:
it would be cleaner to have these 3 fields as a named tuple.

Reply:
I moved them, including the execution context as well, into a `PartitionTaskContext`